### PR TITLE
feat: Move buttons in ModView to right

### PR DESCRIPTION
**Pull request by taskinoz**
Updated the mod view styling to use the space better

image

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
Awesome, thanks for the PR <3

Imma revert the unrelated changes and then let @Alystrasz give it a final look before merge ^^

**Comment by GeckoEidechse** [comment]
Aight, reverted the changes not related to ModView. The changes to release-candidate number conversion sadly didn't work so I also reverted them.

File: src-vue/src/components/LocalModCard.vue [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
+                v-if="mod.thunderstore_mod_string != null"
+                :title="$t('mods.local.part_of_ts_mod') + '\n' + mod.thunderstore_mod_string"
+                src="/src/assets/thunderstore-icon.png"
+                class="image"
+                height="16"
+                style="margin: 0 5px;"
+            />
+        </div>
+        <div>
+            <el-switch style="padding-left: 5px; padding-right: 5px; --el-switch-on-color: #13ce66; --el-switch-off-color: #8957e5" v-model="mod.enabled"
```
Comment (Alystrasz) [inline src-vue/src/components/LocalModCard.vue:15]: Style attribute maybe is a bit too long, you should put that in CSS section as well

**Review by Alystrasz — CHANGES_REQUESTED** [review]
Apart from CSS styles refactoring, this looks good.
Confirmed working in testing.

File: src-vue/src/components/LocalModCard.vue [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
-        {{ mod.name }}
-        <span v-if="mod.version != null">(v{{ mod.version }})</span>
-        <img
-            v-if="mod.thunderstore_mod_string != null"
-            :title="$t('mods.local.part_of_ts_mod') + '\n' + mod.thunderstore_mod_string"
-            src="/src/assets/thunderstore-icon.png"
-            class="image"
-            height="16"
-        />
+    <el-card shadow="hover" :body-style="{display: 'flex', alignItems: 'center', width: '100%', justifyContent: 'space-between'}">
```
[the remark this answers is not in the document]
Comment (taskinoz) [reply src-vue/src/components/LocalModCard.vue]: I've moved the CSS to the Style section

File: src-vue/src/components/LocalModCard.vue (hunk 1 above)
Comment (taskinoz) [reply src-vue/src/components/LocalModCard.vue]: I've moved the CSS to the Style section

**Review by Alystrasz — APPROVED** [review]
Requested changes were addressed.
Confirmed working in testing.

**Pull request merged**